src/test/java/hudson/markup/MyspacePolicyTest.java

```diff
@@ -15,14 +15,14 @@
 public class MyspacePolicyTest extends Assert {
     @Test
     public void testPolicy() {
-        assertIntact("<a href='http://www.cloudbees.com'>CB</a>");
-        assertIntact("<a href='relative/link'>relative</a>");
-        assertIntact("<a href='mailto:kk&#64;kohsuke.org'>myself</a>");
+        assertSanitize("<a href='http://www.cloudbees.com' rel='nofollow'>CB</a>", "<a href='http://www.cloudbees.com'>CB</a>");
+        assertSanitize("<a href='relative/link' rel='nofollow'>relative</a>", "<a href='relative/link'>relative</a>");
+        assertSanitize("<a href='mailto:kk&#64;kohsuke.org' rel='nofollow'>myself</a>", "<a href='mailto:kk&#64;kohsuke.org'>myself</a>");
         assertReject("javascript","<a href='javascript:alert(5)'>test</a>");
 
-        assertIntact("<img src='http://www.cloudbees.com'>");
-        assertIntact("<img src='relative/test.png'>");
-        assertIntact("<img src='relative/test.png'>");
+        assertIntact("<img src='http://www.cloudbees.com' />");
+        assertIntact("<img src='relative/test.png' />");
+        assertIntact("<img src='relative/test.png' />");
```

Comment:
```suggestion
        assertIntact("<img src='relative/test.png' />");
        assertReject("onerror","<img src='x' onerror='alert(5)'>");
```

Tested locally, it passes :)